src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/validation/AuditValidator.java

```diff
@@ -2,12 +2,36 @@
 
 import com.amazon.opendistroforelasticsearch.security.DefaultObjectMapper;
 import com.amazon.opendistroforelasticsearch.security.auditlog.config.AuditConfig;
+import com.amazon.opendistroforelasticsearch.security.auditlog.impl.AuditCategory;
+import com.google.common.collect.ImmutableSet;
 import org.elasticsearch.common.bytes.BytesReference;
 import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.rest.RestRequest;
 
+import java.util.Set;
+
 public class AuditValidator extends AbstractConfigurationValidator {
 
+    private static final Set<AuditCategory> DISABLED_REST_CATEGORIES = ImmutableSet.of(
```

Comment:
why these categories are disable at rest layer? 
Aren't these all valid categories?

Reply:
This is a validation check. They are valid categories for "disabled_rest_categories" field. Similarly for transport